lib/options-handler.js

```diff
@@ -277,7 +282,48 @@ export class ClientOptionsHandler {
         return '9999999.99999.999';
     }
 
-    setCompilers(compilers) {
+    getRemoteId(remoteUrl, language) {
+        const url = new URL(remoteUrl);
+        return url.host.replace(/\./g, '_') + '_' + language;
+    }
```

Comment:
You already added this one lib-utils

Reply:
yes, but that's static frontend code.. i think theres a way to share code between the two, will see if i can do that

Reply:
Oh! I did not realize this was not already the case, I must have confused it for another shared part, sorry!

Reply:
Don't think there is a way to currently do it. I'll keep it as is for now.